sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -256,6 +259,33 @@ private boolean canLogAtLevel(LogLevel logLevel, LogLevel environmentLoggingLeve
         }
     }
 
+    /**
+     * Checking the system log with the preference order of slf4j and environment variable.
+     *
+     * @return The log level.
+     */
+    private void loadSystemLogLevel() {
+        if (rootLogLevel != null) {
+            return;
+        }
+        final StaticLoggerBinder binder = StaticLoggerBinder.getSingleton();
+        if (binder.getLoggerFactory() != null) {
+            if (logger.isDebugEnabled()) {
+                rootLogLevel = LogLevel.VERBOSE;
+            }
+            if (logger.isInfoEnabled()) {
+                rootLogLevel = LogLevel.INFORMATIONAL;
+            }
+            if (logger.isWarnEnabled()) {
+                rootLogLevel = LogLevel.WARNING;
+            }
+            if (logger.isErrorEnabled()) {
+                rootLogLevel = LogLevel.ERROR;
+            }
+        }
```

Comment:
Would this make more sense to go into `canLogAtLevel`? Have that check if there is a binder and if it contains one use the enabled logging levels on it to determine if it can log, if not set check against the environment variables?

Reply:
This call is to return the logLevel from app or sys. Then both canLogAtLevel and performLogging can use the loglevel.